src/anemoi/models/__init__.py

```diff
@@ -6,4 +6,4 @@
 # nor does it submit to any jurisdiction.
 
 
-from ._version import __version__
+from ._version import __version__ as __version__
```

Comment:
Why is this different from the previous version?

Reply:
`ruff` is changing it. 
```
F401 `._version.__version__` imported but unused; consider removing, adding to `__all__`, or using a redundant alias
```